x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/parser/StatementParserTests.java

```diff
@@ -338,17 +338,17 @@ public void testInlineStatsWithoutGroups() {
     }
 
     public void testIdentifiersAsIndexPattern() {
-        assertIdentifierAsIndexPattern("foo", "from `foo`");
-        assertIdentifierAsIndexPattern("foo,test-*", "from `foo`,`test-*`");
+        assertIdentifierAsIndexPattern("foo", "from \"foo\"");
+        assertIdentifierAsIndexPattern("foo,test-*", "from \"foo\",\"test-*\" ");
         assertIdentifierAsIndexPattern("foo,test-*", "from foo,test-*");
         assertIdentifierAsIndexPattern("123-test@foo_bar+baz1", "from 123-test@foo_bar+baz1");
-        assertIdentifierAsIndexPattern("foo,test-*,abc", "from `foo`,`test-*`,abc");
-        assertIdentifierAsIndexPattern("foo,     test-*, abc, xyz", "from `foo,     test-*, abc, xyz`");
-        assertIdentifierAsIndexPattern("foo,     test-*, abc, xyz,test123", "from `foo,     test-*, abc, xyz`,     test123");
+        assertIdentifierAsIndexPattern("foo,test-*,abc", "from \"\"\"foo\"\"\",\"\"\"test-*\"\"\",abc");
+        assertIdentifierAsIndexPattern("foo,     test-*, abc, xyz", "from \"foo,     test-*, abc, xyz\"");
+        assertIdentifierAsIndexPattern("foo,     test-*, abc, xyz,test123", "from \"\"\"foo,     test-*, abc, xyz\"\"\",     test123");
         assertIdentifierAsIndexPattern("foo,test,xyz", "from foo,   test,xyz");
         assertIdentifierAsIndexPattern(
             "<logstash-{now/M{yyyy.MM}}>,<logstash-{now/d{yyyy.MM.dd|+12:00}}>",
-            "from <logstash-{now/M{yyyy.MM}}>, `<logstash-{now/d{yyyy.MM.dd|+12:00}}>`"
+            "from <logstash-{now/M{yyyy.MM}}>, \"<logstash-{now/d{yyyy.MM.dd|+12:00}}>\""
         );
```

Comment:
I am surprised we don't need to update any csv tests - I guess we should add one or two to ensure this change also works correctly end-to-end.

Reply:
And use these csv tests in the docs for `FROM`; currently, I think they contain incorrect syntax (missing backticks/quotes):
![image](https://github.com/elastic/elasticsearch/assets/88324834/4b66d025-5dfc-4660-b4d0-2efb7993e579)